### PR TITLE
Add etched runes in block

### PR DESCRIPTION
**Pull request by lugondev**
Issue:  #3352

in reviewing

### CODE REVIEW

File: templates/block.html [hunk 1]
```diff
@@ -21,6 +21,14 @@ <h1>Block {{ self.height }}</h1>
 next
 %% }
 </div>
+<h2>{{"Rune".tally(self.runes.len())}}</h2>
+%% if self.runes.len() > 0 {
+<ul>
+  %% for entry in &self.runes {
+  <li><a href=/rune/{{ entry.spaced_rune }}>{{ entry.spaced_rune }}</a></li>
```
Comment (raphjaph) [inline templates/block.html:28]: Let's put Runes beneath inscriptions

File: src/index.rs [hunk 2]
```diff
@@ -1729,6 +1729,29 @@ impl Index {
...
+      tx: 0,
+    };
+
+    let max_id = RuneId {
+      block: u64::from(block_height + 1),
+      tx: 0,
+    };
+
+    let runes = rune_id_to_rune_entry
+      .range(min_id.store()..max_id.store())?
```
Comment (raphjaph) [inline src/index.rs:1748]: could you add a test, that checks whether the next blocks first rune is not included

**Review by raphjaph — CHANGES_REQUESTED** [review]
Nice PR!

One more test as mentioned would be good.

File: src/index.rs (hunk 2 above)
Comment (lugondev) [reply src/index.rs:1748]: Will do it

**Comment by lugondev** [comment]
yeh, I'm doing

**Comment by lugondev** [comment]
@raphjaph finish test case!

**Review by raphjaph — APPROVED** [review]
LGTM

**Pull request merged**